app/resources/static/forms.js.template

```diff
@@ -367,22 +367,25 @@ function validate_fields() {
 }
 
 // Make it possible to expand the form to see the detail form
-function make_form_expandable(expandedSection) {
-  var expandedPart = jQuery(expandedSection).find("div.expanded-part");
-  var expandButton = jQuery(expandedSection).find("a.expand-button");
-
-  expandedPart.hide();
-  expandButton.click(function() {
-    if (expandedPart.is(":visible")) {
-      expandedPart.slideUp("slow");  
-      expandButton.removeClass(
-        'expand-button-icon-up').addClass('expand-button-icon-down');
+function make_form_expandable(expandedSectionSelector) {
```

Comment:
Have you confirmed that this part works fine after your change?

Reply:
Yes.